scholia/app/templates/author-curation_missing-citing-author-items.sparql

```diff
@@ -29,7 +29,7 @@ WITH {
 WHERE {
   INCLUDE %result
   # Label the results
-  SERVICE wikibase:label { bd:serviceParam wikibase:language "en,da,de,es,fr,jp,nl,no,ru,sv,zh". }
+  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en" . }
```

Comment:
What about fallback, e.g., de label define but not others?

Reply:
I think with `[AUTO_LANGUAGE],en` it can indeed only fallback to English.

Reply:
The issue is much more fallback, e.g., you can experience many Czech book items with only Czech labels. For me, they appear as QID, while they could fall back on cz. The [AUTO_LANGAUGE],mul,en,cz,da,de,fr,jp,nb,sv,uk,zh,etc would be an attempt to do this.

Reply:
@fnielsen, I am pondering about this... maybe we should file a phabricator request to have something like `[ANY_LANGUAGE]' ? Because we cannot really list all possible languages and cannot predict which language will have the one single label, rigth?

Reply:
Yes.

Reply:
https://phabricator.wikimedia.org/T377654